recipe/meta.yaml

```diff
@@ -1,53 +1,142 @@
+        - pyproj >=1.9.3,<3.4
+        - pyshp >=1.2.0,<2.2
+        - six >=1.10,<1.16
+    test:
+      imports:
+        - mpl_toolkits.basemap
+      requires:
+        - pip
+        - {{ pin_subpackage('basemap-data') }}
+      commands:
+        - pip check
+        - python -c "from mpl_toolkits.basemap import Basemap"
+    about:
+      home: https://matplotlib.org/basemap
+      license: MIT
+      license_family: MIT
+      license_file:
+        - packages/basemap/LICENSE
+      summary: 'Plot on map projections using matplotlib'
+      description: |
+        Basemap toolkit is a library for plotting 2D data on maps in Python.
+        It is similar in functionality to the matlab mapping toolbox, the IDL mapping facilities, GrADS, or the Generic Mapping Tools.
+      dev_url: https://github.com/matplotlib/basemap
+      doc_url: https://matplotlib.org/basemap/
+      doc_source_url: https://github.com/matplotlib/basemap/blob/v{{ version }}/packages/basemap/doc/index.rst
+
+  - name: basemap-data
+    script: build-basemap.sh  # [not win]
+    script: bld-basemap.bat   # [win]
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python >=3.6
+    about:
+      home: https://matplotlib.org/basemap
+      license: LGPL-3.0-or-later
+      license_family: LGPL
+      license_file: packages/basemap_data_hires/COPYING.LESSER
+      summary: Plot on map projections (with coastlines and political boundaries) using matplotlib.
+      dev_url: https://github.com/matplotlib/basemap
+      doc_url: https://matplotlib.org/basemap/
+      doc_source_url: https://github.com/matplotlib/basemap/blob/v{{ version }}/packages/basemap/doc/index.rst
+
+  - name: basemap-data-hires
+    script: build-basemap.sh  # [not win]
+    script: bld-basemap.bat   # [win]
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - {{ pin_subpackage('basemap-data') }}
+        - python >=3.6
+    about:
+      home: https://matplotlib.org/basemap
+      license: LGPL-3.0-or-later
+      license_family: LGPL
+      license_file: packages/basemap_data_hires/COPYING.LESSER
+      summary: Plot on map projections (with coastlines and political boundaries) using matplotlib.
+      dev_url: https://github.com/matplotlib/basemap
+      doc_url: https://matplotlib.org/basemap/
+      doc_source_url: https://github.com/matplotlib/basemap/blob/v{{ version }}/packages/basemap/doc/index.rst
+
+
 
 about:
-  home: http://matplotlib.org/basemap
-  license: MIT
-  license_file: README.md
-  summary: 'Plot on map projections using matplotlib'
+  home: https://matplotlib.org/basemap
+  license: LGPL-3.0-or-later
+  license_family: LGPL
+  summary: split for basemap.
 
 extra:
   recipe-maintainers:
     - molinav
     - ocefpaf
-    - ReimarBauer
```

Comment:
Here you get removed from the recipe-maintainers list. I guess this is not intended, right?

Reply:
yes, It was because of the initial copy, and currently Draft ;)